required/latex-lab/latex-lab-math.dtx

```diff
@@ -335,16 +349,22 @@
         $$
       }
   }
-
+%    \end{macrocode}
+% To allow to use the code without tagging we guard. But probably tagpdf should
```

Comment:
Looks good; but I have not checked whether this is the only place in the code where such a guard is needed.